tests/PHPStan/Analyser/nsrt/bug-12297.php

```diff
@@ -0,0 +1,19 @@
+<?php // lint >= 7.4
```

Comment:
What happens before 7.4?

Reply:
no difference. I had the impression the nsrt/ tests require a lint php condition

Reply:
They don't. If it's omitted then it's always analysed.

I'm asking because after merging to 2.0.x I'd have to deleted it anyway. But if the tests passes on 7.2+ please remove this comment.